src/NuGet.Core/NuGet.Commands/Strings.resx

```diff
@@ -1107,4 +1107,8 @@ Non-HTTPS access will be removed in a future version. Consider migrating to 'HTT
     <value>You are running the '{0}' operation with an 'HTTP' source: {1}. NuGet requires HTTPS sources. To use an HTTP source, you must explicitly set 'allowInsecureConnections' to true in your NuGet.Config file. Please refer to https://aka.ms/nuget-https-everywhere.</value>
     <comment>0 - The command name. Ex. Push/Delete. 1 - server URI.</comment>
   </data>
+  <data name="Error_HttpSource_Single_Short" xml:space="preserve">
```

Comment:
Can we the existing `Error_HttpSource_Single` resource instead of adding a new one?

Reply:
It is because they are a little different. One tells the user to configure `allowInsecureConnections` while the other one does not.

Reply:
There is duplication in the strings for `Error_HttpSource_Single`, `Error_HttpSources_Multiple` and `Error_HttpSource_Single_Short` resources. If we ever have to change the error message, we have to update in all the places. I guess we should be okay good for now.